src/client.ts

```diff
@@ -100,7 +100,7 @@ export const createClient = (args: AuthProps, opts?: ConstructorOpts) => {
     const method = init ? init.method || 'GET' : 'GET'
 
     if (opts?.log) {
-      const message = `[${method}] ${url} ${init ? init.body : ''}`
+      const message = `[${method}] ${url}`
```

Comment:
maybe we could include something like
```suggestion
      const message = `[${method}] ${url} ${{opts?.logFull && init ? init.body : ''}`
```

where opts.logFull is `false` by default that way we can at least optionally turn it on if we REALLY need to

Reply:
Added!